.gitignore

```diff
@@ -7,3 +7,5 @@ build/
 dist/
 .idea/
 venv/
+.vscode/
+.coverage
```

Comment:
Why we have to add `.coverage` here?

Reply:
It gets created whenever I run presubmit checks locally.

> $ ./run_presubmit.sh